moveit_core/kinematic_constraints/include/moveit/kinematic_constraints/kinematic_constraint.h

```diff
@@ -478,6 +478,11 @@ class OrientationConstraint : public KinematicConstraint
     return absolute_z_axis_tolerance_;
   }
 
+  int getParameterization() const
```

Comment:
Can we use the term `parameterization_type_`, since the brief also refers to it like that? I know the message doesn't, but it seems clearer to me.

Reply:
I agree!